src/main/scala/strawman/collection/Iterable.scala

```diff
@@ -770,6 +770,12 @@ trait IterableOps[+A, +CC[X], +C] extends Any {
       else View.Empty
     }
 
+  /** Alias for `appendAll` */
+  @`inline` final def concat[B >: A](suffix: Iterable[B]): CC[B] = appendAll(suffix)
+
+  /** Alias for `concat` */
+  @`inline` final def ++ [B >: A](suffix: Iterable[B]): CC[B] = concat(suffix)
```

Comment:
Don't have double-indirection if you don't need it.  If the "real" method is called `appendAll`, then `++` should forward directly to it.